src/encoded/commands/extract_test_data.py

```diff
@@ -122,7 +122,7 @@ def anon_pipeline():
 
 
 def run(pipeline, inpath, outpath):
-    for item_type in ORDER:
+    for item_type in ORDER():
```

Comment:
I've made comments about this change to ORDER on other PRs. I really don't like this change.

Reply:
Oh shoot, I made that change in snovault, and in smaht-portal, but forgot to propagate to fourfront - will do - thanks.